server/src/main/java/org/opensearch/index/translog/InternalTranslogManager.java

```diff
@@ -289,6 +289,11 @@ public void ensureCanFlush() {
         }
     }
 
+    @Override
+    public void minSeqNoRequired(long seqNo) {
```

Comment:
This methods seem to be no-op everywhere except the RemoteFsTranslog. Is there a way we could fetch the `lastRefreshedCheckpoint`directly?

Reply:
Good point Ashish . We can't rely on `lastRefreshedCheckpoint` as even if remote segment upload fails, it is going to get incremented .